runbotics/runbotics-desktop/src/action/microsoft/excel/excel.service.ts

```diff
@@ -179,6 +179,20 @@ export class ExcelService {
             );
     }
 
+    /**
+     * @see https://learn.microsoft.com/en-us/graph/api/range-delete?view=graph-rest-1.0&tabs=http
+     */
+    deleteColumns(session: ExcelSession, columnRange: string) {
+        return this.microsoftGraphService.
+            post(
+                this.createWorkbookUrl(session, `/worksheets/${session.worksheetName}/range(address='${columnRange}')/delete`),
+                {
+                    ...this.getSessionHeader(session),
+                    shift: 'left'
```

Comment:
what is it for?

Reply:
This function sends request to delete column/columns in worksheet. You can read more here: https://learn.microsoft.com/en-us/graph/api/range-delete?view=graph-rest-1.0&tabs=http